pyaedt/hfss.py

```diff
@@ -2326,10 +2326,23 @@ def validate_full_design(self, dname=None, outputdir=None, ports=None):
 
         Returns
         -------
-        list
-            List of all the validation information for later use.
+        tuple
+            Tuple containing a list of all the validation information for later use
+            and a bool indicating if the validation was successful or not.
```

Comment:
```suggestion
        list
            List of all the validation information for later use.
        bool
            Indicates if the validation was successful or not.
```

Would prefer if detail the items we return separately.